app/utils/ajax-helpers.ts

```diff
@@ -30,5 +31,5 @@ export default function authenticatedAJAX(options) {
             },
         });
     }
-    return $.ajax(options);
+    return new EmberPromise((resolve, reject) => $.ajax(options).then(resolve, reject));
```

Comment:
What does this do? Is wrapping it in an RSVP promise necessary?

Reply:
See: https://github.com/CenterForOpenScience/ember-osf-web/pull/179/files#diff-61cf47780bd3aa2cfdc9e00027572539L74

This handles that generally.